copier/copier.go

```diff
@@ -1149,6 +1149,10 @@ func copierHandlerGet(bulkWriter io.Writer, req request, pm *fileutils.PatternMa
 						}
 						symlinkTarget = target
 					}
+					if info.Mode()&os.ModeType == os.ModeSocket {
+						logrus.Warningf("Skipping socket %q", info.Name())
```

Comment:
Can we use `Debugf` instead?  I am afraid this could cause some noise.

Reply:
I believe it should stay a warning. Users should be told that sockets will not be recorded within the image.  Users should actually only put sockets in tmpfs but that is a different story.  I don't think this is a common pattern.

Reply:
If it becomes noisy we can always revisit the situation.

Reply:
Sounds good to me.

It may be helpful to add more information about the SLOC to the log:
```golang
logrus.Warningf("buildah/copier: skipping socket %q", info.Name())
```
?

Reply:
Sure.

Reply:
switched to `Debugf`